config/checkstyle-input-suppressions.xml

```diff
@@ -1605,18 +1605,6 @@
             files="checks[\\/]annotation[\\/]annotationlocation[\\/]InputAnnotationLocationWithoutAnnotations.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]annotation[\\/]annotationlocation[\\/]inputs[\\/]PackageAnnotations.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>
```

Comment:
>proof that all suppressions are removed :

Please explain why `grep` shows two results and has an exit code of zero if all suppressions are removed.